packager_test.go

```diff
@@ -233,3 +233,42 @@ func TestNoExchanges(t *testing.T) {
 		})
 	}
 }
+
+func TestSubresourceErrors(t *testing.T) {
+	handlers := http.NewServeMux()
+	handlers.Handle(
+		"example.org/hello.html",
+		stubHTMLHandler(`<!doctype html>`+
+			`<link href="foo.css" rel="stylesheet">`+
```

Comment:
optional: rename to more descriptive names e.g. foo -> valid, bar -> invalid-1 and baz -> invalid-2 or something.